ewlegal.cls

```diff
@@ -0,0 +1,84 @@
+%% TODO Stock beginning text, copyrights and project info and all that
+
+%% Work consists of the files engwales-legal.cls and accompanying engwales-*.sty
+
+
+\NeedsTeXFormat{LaTeX2e}
+\ProvidesClass{ewlegal}[2021-02-21 Legal Templates for England and Wales]
+
+%% Uses article as a base
+\LoadClass[12pt,a4paper]{article}% TODO Perhaps use KOMA-Script?
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Options
+%%%%%
+
+%% Easy option management
+\RequirePackage{kvoptions}
+
+%% Select the template used
+% TODO Be able to nicely handle multiple templates being selected
+
+\DeclareBoolOption[false]{pleading}
+
+\DeclareBoolOption[false]{opinion}
+
+
+%% Extras
+
+
+
+%% Process options given by the user
+\ProcessKeyvalOptions*
+
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%% Packages
+%%%%%
+
+%% Generic Packages
+
+%% Easy formatting of section titles
+\RequirePackage[explicit]{titlesec}
+
+%% Allows for easy use of logic
+\RequirePackage{etoolbox}
+
+%% Fancy headers and footers
+\RequirePackage{fancyhdr}
+
+%% Spaced out boxes
+\RequirePackage{soul}
+% Doesn't seem to be reccommended anymore due to issues with luatex
```

Comment:
do we want to implement our own version of spaced out boxes then? if that's the idea change this to TODO?

Reply:
I couldn't see why we needed spaced out boxes to begin with. Though I'll see if I can find a different solution.

Reply:
it's used in `\so{BETWEEN}`. of course worst scenario we could just hard code the BETWEEN text if that's the only place where this is needed. in any case i think we can resolve this in other PRs?

Reply:
Yeah, that's why I left the todo to remind me for a later PR. I'm going to resolve this then.

Reply:
this is not marked as TODO tho?

Reply:
Added one there.